src/cadenza.js

```diff
@@ -154,26 +154,29 @@ export class CadenzaClient {
    * @param {boolean} [options.debug]
    */
   constructor(baseUrl, { debug = false, iframe, webApplication } = {}) {
-    assert(validUrl(baseUrl), `Invalid baseUrl: ${baseUrl}`);
     if (webApplication) {
       assert(
         validExternalLinkKey(webApplication),
         `Invalid webApplication parameter: ${webApplication}`,
       );
     }
 
-    // Remove trailing /
-    if (baseUrl.at(-1) === '/') {
-      baseUrl = baseUrl.substring(0, baseUrl.length - 1);
+    if (baseUrl !== undefined) {
```

Comment:
I'd rather not be so strict about null vs undefined. People might not be aware of the difference and not use TS.

Proposal: baseUrl != null